Sources/Rendering/Core/Camera/index.d.ts

```diff
@@ -0,0 +1,302 @@
+import VtkObject from '../../../macro';
+
+export interface vtkCamera extends VtkObject {
+    /**
+     * 
+     */
+    orthogonalizeViewUp(): void;
+
+    /**
+     * 
+     * @param x 
+     * @param y 
+     * @param z 
+     */
+    setPosition(x: number, y: number, z: number): void;
+
+    /**
+     * 
+     * @param x 
+     * @param y 
+     * @param z 
+     */
+    setFocalPoint(x: any, y: any, z: any): void;
```

Comment:
should they be number too? 

Also those methods can support both `xyz[]` and `x, y, z`

Reply:
Yes i'll do another pass for the param and the setter/getter

Reply:
Also more of those get generated. (Look at [setGetArray](https://github.com/Kitware/vtk-js/blob/master/Sources/macro.js#L484-L485) macro to see what I mean).